tox.ini

```diff
@@ -10,9 +10,6 @@ envlist=
 max-line-length= 100
 exclude=
 ignore =
-    # E252 missing whitespace around parameter equals
-    # Already used too much in the codebase at the point of introduction.
-    E252,
     # W503 line break before binary operator
     # It's either this or W504 (line break _after_ binary operator), pick your poison.
     # W503 gets enabled when E252 gets ignored, see:
```

Comment:
W503 was only here because of E252, so we can actually delete the W503 ignore now, also.